README.md

```diff
@@ -508,46 +434,56 @@ $ make toolchains
 * `repo sync` can take an additional parameter -j to sync multiple remotes. For
    example `repo sync -j3` will sync three remotes in parallel.
 
-### 6.3. QEMU
-After getting the source and toolchain, just run:
+---
+### 5.3. QEMU
+After getting the source and toolchain, just run (from the `build` folder)
 ```
 $ make all run
 ```
 and everything should compile and at the end QEMU should start.
 
-### 6.4. FVP
-After getting the source and toolchain you must also get the get Foundation
-Model
+---
+### 5.4. FVP
+After getting the source and toolchain you must also obtain Foundation Model
 ([link](http://www.arm.com/products/tools/models/fast-models/foundation-model.php))
-and untar it to the forest root, then just run:
+binaries and untar it to the forest root, then just run (from the `build` folder)
+
 ```
 $ make all run
 ```
 and everything should compile and at the end FVP should start.
 
-### 6.5. Hikey
-After running `make` above, follow the instructions at
-[flash-binaries-to-emmc](https://github.com/96boards/documentation/wiki/HiKeyUEFI#flash-binaries-to-emmc-)
-to flash all the required images to and boot the board.
+---
+### 5.5. HiKey
+After getting the source and toolchain, just run (from the `build` folder)
```

Comment:
@jbech-linaro yes, 2.4.1 is old so indeed chances are low that people would stumble on this version issue. I did, because Travis CI still runs on 12.04 LTS...